servicemesh-install/topics/install.adoc

```diff
@@ -27,301 +71,572 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
+      # Your GitHub username
+      LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME:
+      # Your GitHub Mission Control access token
+      LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN:
+
+    threeScale:
+      enabled: false
+
+    istio:
+      global:
+        proxy:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 128Mi
+            limits:
+              cpu: 500m
+              memory: 128Mi
+
+      gateways:
+        istio-egressgateway:
+          autoscaleEnabled: false
+        istio-ingressgateway:
+          autoscaleEnabled: false
+          ior_enabled: false
+
+      mixer:
+        policy:
+          autoscaleEnabled: false
+
+        telemetry:
+          autoscaleEnabled: false
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1G
+            limits:
+              cpu: 500m
+              memory: 4G
+
+      pilot:
+        autoscaleEnabled: false
+        traceSampling: 100.0
+
+      kiali:
+       dashboard:
+          user: admin
+          passphrase: admin
+      tracing:
+        enabled: true
 ----
 
-The following example illustrates the minimum required to install the control plane.  This minimal example custom resource deploys the CentOS-based community Istio images.
+[[custom-resource-parameters]]
+== Custom resource parameters
+
+The following examples illustrate use of the supported custom resource parameters for {ProductName} and the tables provide additional information about supported parameters.
+
+[IMPORTANT]
+====
+The resources you configure for {ProductName} with these custom resource parameters, including CPUs, memory, and the number of pods, are based on the configuration of your OpenShift cluster. Configure these parameters based on the available resources in your current cluster configuration.
+====
+
+=== Istio global example
 
-.Minimum example istio-installation.yaml
+[NOTE]
+====
+In order for the 3scale Istio Adapter to work, `disablePolicyChecks` must be `false`.
+====
 
 [source,yaml]
 ----
-apiVersion: "istio.openshift.com/v1alpha1"
-kind: "Installation"
-metadata:
-  name: "istio-installation"
+  istio:
+    global:
+      hub: `maistra/` or `openshift-istio-tech-preview/`
+      tag: 0.10.0
+      proxy:
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 500m
+            memory: 128Mi
+      mtls: false
+      disablePolicyChecks: true
+      policyCheckFailOpen: false
+      imagePullSecrets:
+        - MyPullSecret
 ----
 
-[[custom-resource-parameters]]
-== Custom resource parameters
-
-The following tables list the supported custom resource parameters for {ProductName}.
+[NOTE]
+====
+See the OpenShift documentation on https://docs.openshift.com/enterprise/3.1/dev_guide/compute_resources.html[Compute Resources] for additional details on specifying CPU and memory resources for the containers in your pod.
```

Comment:
@geekspertise, did you mean to link to 3.11 instead of 3.1?

Reply:
@kalexand-rh I updated the link and updated the PR.